### PR TITLE
Launch 23.3

### PR DESCRIPTION
**Pull request by Deflaimun**
Previews:
57 new pages
141 modified pages

### Cloud 
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview

### Deploy
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/kubernetes/local-guide
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/manual/production/production-deployment-automation
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/deploy/index

### Develop
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/develop/data-transforms/how-transforms-work
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/develop/data-transforms/index
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/develop/data-transforms/k-run-transforms
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/develop/data-transforms/run-transforms-index
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/develop/data-transforms/run-transforms
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/develop/produce-data/configure-producers

### Get started
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/get-started/whats-new

### Manage
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/audit-logging
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/audit-logging/audit-log-samples
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/cluster-maintenance/cluster-balancing
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/cluster-maintenance/compaction-settings
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/cluster-maintenance/disk-utilization
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/kubernetes/decommission-brokers
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/kubernetes/k-recovery-mode
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/kubernetes/k-whole-cluster-restore
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/kubernetes/security/authentication/sasl-kubernetes
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/kubernetes/security/tls/index
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/recovery-mode
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/schema-reg/index
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/schema-reg/schema-id-validation
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/schema-reg/schema-reg-api
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/schema-reg/schema-reg-overview
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/schema-reg/schema-reg-ui
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/security/authentication
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/security/encryption
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/manage/whole-cluster-restore

### Upgrade
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/upgrade/deprecated/index
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/upgrade/migrate/kubernetes/operator

### Reference
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/api-reference
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/cluster-properties
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/console/programmable-push-filters
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/console/record-deserialization
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/data-transform-api
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/internal-metrics-reference
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/node-properties
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/public-metrics-reference

#### rpk
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-partitions-unsafe-recover
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-acl/rpk-acl-user-create
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-acl/rpk-acl-user-delete
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-acl/rpk-acl-user-list
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-acl/rpk-acl-user-update
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-acl/rpk-acl-user
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-config-set
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-partitions-disable
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-partitions-enable
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-partitions-list
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-partitions-move-cancel
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-partitions-move-status
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-partitions-move
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-storage-restore-start
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-storage-restore-status
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-storage-restore
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-txn-describe-producers
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-txn-describe
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-txn-list
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-cluster/rpk-cluster-txn
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-container/rpk-container-start
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-debug/rpk-debug-bundle
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-profile/rpk-profile-create
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-redpanda/rpk-redpanda-config-set
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-registry/rpk-registry-compatibility-level-set
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-registry/rpk-registry-schema-create
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-topic/rpk-topic-consume
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-topic/rpk-topic-produce
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-transform/rpk-transform-deploy
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-transform/rpk-transform-list
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-transform/rpk-transform
https://deploy-preview-205--redpanda-docs-preview.netlify.app/current/reference/rpk/rpk-version

### CODE REVIEW

**Comment by Deflaimun** [comment]
Needs to fix some of the build warnings before merging

2:50:36 AM: [05:50:36.147] WARN (asciidoctor): skipping reference to missing attribute: link-cluster-restore
2:50:36 AM:     file: /opt/build/repo/modules/manage/pages/whole-cluster-restore.adoc
2:50:36 AM:     source: /opt/build/repo (branch: HEAD <worktree>)
2:50:38 AM: [05:50:38.048] WARN (asciidoctor): skipping reference to missing attribute: link-cluster-restore
2:50:38 AM:     file: /opt/build/repo/modules/manage/pages/kubernetes/k-whole-cluster-restore.adoc
2:50:38 AM:     source: /opt/build/repo (branch: HEAD <worktree>)
2:50:38 AM: [05:50:38.518] ERROR (asciidoctor): target of xref not found: reference:redpanda-operator/index.adoc
2:50:38 AM:     file: /opt/build/repo/modules/upgrade/pages/deprecated/index.adoc
2:50:38 AM:     source: /opt/build/repo (branch: HEAD <worktree>)
2:50:42 AM: [05:50:42.504] WARN (asciidoctor): unterminated open block
2:50:42 AM:     file: /opt/build/repo/modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment-automation.adoc:169
2:50:42 AM:     source: /opt/build/repo (branch: HEAD <worktree>)
2:50:42 AM: [05:50:42.505] WARN (asciidoctor): unterminated example block
2:50:42 AM:     file: /opt/build/repo/modules/deploy/pages/deployment-option/self-hosted/manual/production/production-deployment-automation.adoc:170
2:50:42 AM:     source: /opt/build/repo (branch: HEAD <worktree>)
2:50:59 AM: [05:50:59.902] WARN (unlisted-pages-extension): detected unlisted page
2:50:59 AM:     source: /opt/build/repo (branch: HEAD <worktree>)
2:50:59 AM:     file: /opt/build/repo/modules/manage/pages/audit-logging/audit-log-samples.adoc
2:50:59 AM: [05:50:59.903] WARN (unlisted-pages-extension): detected unlisted page
2:50:59 AM:     source: /opt/build/repo (branch: HEAD <worktree>)
2:50:59 AM:     file: /opt/build/repo/modules/deploy/pages/deployment-option/cloud/provision-a-dedicated-cluster/index.adoc
2:50:59 AM: [05:50:59.905] WARN (unlisted-pages-extension): detected unlisted page
2:50:59 AM:     source: https://github.com/redpanda-data/documentation (branch: main)
2:50:59 AM:     file: modules/labs/pages/install-preview.adoc
2:50:59 AM: [05:50:59.906] WARN (unlisted-pages-extension): detected unlisted page
2:50:59 AM:     source: https://github.com/redpanda-data/documentation (branch: main)
2:50:59 AM:     file: modules/deploy/pages/deployment-option/cloud/provision-a-dedicated-cluster/index.adoc

File: antora.yml [hunk 1]
```diff
@@ -1,12 +1,14 @@
 name: ROOT
 title: Redpanda
-version: 23.2
+version: 23.3
+display_version: '23.3 Beta'
+prerelease: true
```
Comment (JakeSCahill) [inline antora.yml]:
```suggestion
```

File: antora.yml [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 start_page: get-started:intro-to-events.adoc
 nav:
 - modules/ROOT/nav.adoc
 asciidoc:
   attributes:
-    full-version: 23.2.3
+    full-version: 23.3.0
```
Comment (JakeSCahill) [inline antora.yml]:
```suggestion
    full-version: 23.3.1
```

File: local-antora-playbook.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ content:
   - url: .
     branches: HEAD
   - url: https://github.com/redpanda-data/documentation
-    branches: [v/*, '!v/23.2', api, shared, site-search,v-WIP/*]
+    branches: [v/*, main,api, shared, site-search]
```
Comment (JakeSCahill) [inline local-antora-playbook.yml]:
```suggestion
    branches: [v/*, api, shared, site-search]
```

**Review by JakeSCahill — COMMENTED** [review]
Updated a few configs required for release

**Review by greg-szrama — APPROVED** [review]

**Review by Feediver1 — APPROVED** [review]

**Pull request merged**